plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/edit/struct/SQLTableColumnManager.java

```diff
@@ -90,29 +90,7 @@
         NullNotNullModifier.appendModifier(monitor, column, sql, command);
     };
 
-    protected final ColumnModifier<OBJECT_TYPE> DefaultModifier = (monitor, column, sql, command) -> {
-        String defaultValue = CommonUtils.toString(column.getDefaultValue());
-        if (!CommonUtils.isEmpty(defaultValue)) {
-            DBPDataKind dataKind = column.getDataKind();
-            boolean useQuotes = false;//dataKind == DBPDataKind.STRING;
-            if (!defaultValue.startsWith(QUOTE) && !defaultValue.endsWith(QUOTE)) {
-                if (useQuotes && defaultValue.trim().startsWith(QUOTE)) {
-                    useQuotes = false;
-                }
-                if (dataKind == DBPDataKind.DATETIME) {
-                    final char firstChar = defaultValue.trim().charAt(0);
-                    if (!Character.isLetter(firstChar) && firstChar != '(' && firstChar != '[') {
-                        useQuotes = true;
-                    }
-                }
-            }
-
-            sql.append(" DEFAULT "); //$NON-NLS-1$
-            if (useQuotes) sql.append(QUOTE);
-            sql.append(defaultValue);
-            if (useQuotes) sql.append(QUOTE);
-        }
-    };
+    protected ColumnModifier<OBJECT_TYPE> DefaultModifier = new BaseDefaultModifier();
```

Comment:
```suggestion
    protected ColumnModifier<OBJECT_TYPE> defaultModifier = new BaseDefaultModifier();
```

Reply:
It also should be `final`. Maybe we'll keep it's original name?

Reply:
I'm not insist.